src/routes/analytics/DashboardPage.tsx

```diff
@@ -10,7 +12,13 @@ export function DashboardPage(props: Props) {
 
   return (
     <Box mih="100vh" className="dashboard-container smartscalar">
-      <InteractiveDashboard dashboardId={dashboardId} withTitle withDownloads />
+      <RemountOnThemeChange>
+        <InteractiveDashboard
+          dashboardId={dashboardId}
+          withTitle
+          withDownloads
+        />
+      </RemountOnThemeChange>
```

Comment:
Maintains the illusion of switching between themes, without forcing the entire page to go into a loading spinner.